api-call.py

```diff
@@ -1,29 +1,226 @@
-import requests
+import datetime
 import json
 
+import pytz
+import requests
+import yaml
+
+
+def load_from_config():
+    """Loading important parameters from configuration file
+
+    :returns: The config object
+    """
+
+    with open('configuration.yaml', 'r') as file:
+        return yaml.safe_load(file)
+
+
+def log_error(error_message):
+    """Logging any error messages in the file and appending
+    with date time information in PST
+
+    :param error_message: The error message to log in file comes as a string
+    """
+
+    # Get the current datetime in PST
+    pst_timezone = pytz.timezone('America/Los_Angeles')
+    current_time = datetime.datetime.now(pst_timezone)
+    formatted_message = f"[{current_time}] {error_message}"
```

Comment:
Based on our style guide, we should use single quotes for strings when possible. https://github.com/osu-mist/style-guides/blob/master/Python.md#quoting-strings

Reply:
done.